### ISSUE TITLE
RPM SPEC version does not reflect the current release

### ISSUE THREAD

**Issue by rinigus** [mer-hybris/qt5-qpa-hwcomposer-plugin#89]
For better support of tbuilder, please adjust RPM SPEC version with every release.

**Comment by Thaodan** [comment]
Please use the git tag,  thank you.

**Comment by rinigus** [comment]
Indeed, for many packages git tags can be used. However, in some cases that I have had with tbuilder, git tags and calc of version from them failed with some specific modifications of the submodules. 

Irrespective to the use of tags, it is expected that version string is adjusted in SPEC as well as a part of the release. This repository is one of the few in Sony Tama port that does not follow this expected convention. Please fix it.

**Comment by Thaodan** [comment]
>, in some cases that I have had with tbuilder, git tags and calc of version from them failed with some specific modifications of the >submodules.
Which failures please describe?

In any case the version is to be set to the git tag. There will be no modifications to fix some specific tooling.

**Comment by rinigus** [comment]
> Which failures please describe?

I don't remember. Next time I get something like I had, I will let you know. 

> In any case the version is to be set to the git tag. There will be no modifications to fix some specific tooling.

That's what I am asking for. To keep RPM SPEC and git tags in sync, as a part of the release. As expected when you provide RPM SPEC as a part of the repository.

**Comment by Thaodan** [comment]
> That's what I am asking for. To keep RPM SPEC and git tags in sync, as a part of the release. As expected when you provide RPM SPEC as a part of the repository.
Exactly that will not happen as it will be overridden during build. This is the reason why things like release are set to 0.

**Comment by rinigus** [comment]
Let me be specific:

I am asking to sync line https://github.com/mer-hybris/qt5-qpa-hwcomposer-plugin/blob/master/rpm/qt5-qpa-hwcomposer-plugin.spec#L3 to be the same as the corresponding release.

Right now, this line should be

```
Version:    5.6.2.18
```

Instead, it sets version to 5.6.2.1 and, below, release to 1.

**Comment by martyone** [comment]
> Indeed, for many packages git tags can be used. However, in some cases that I have had with tbuilder, git tags and calc of version from them failed with some specific modifications of the submodules.

I guess it would help you if there was an option to ignore possible local changes and only consider the latest git tag. Could you please file a bug on forum.sailfishos.org for that?
